app/assets/css/main.css

```
@@ -0,0 +1,289 @@
+@import 'tailwindcss';
+@import '@nuxt/ui';
+@import 'tw-animate-css';
+@import './keyframes.css';
+@plugin "@tailwindcss/typography";
+
+@plugin 'daisyui' {
+	root: ':root';
+}
+
+@plugin "daisyui/theme" {
+	name: 'light';
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)